src/main/python/ttconv/tt.py

```diff
@@ -225,6 +270,11 @@ def validate(args):
   print(args.input)
   LOGGER.info("Input file is %s", args.input)
 
+# Ensure that the handler is added only once/globally
+# Otherwise the handler will be called multiple times
+progress = ProgressConsoleHandler()
+LOGGER.addHandler(progress)
```

Comment:
``
```suggestion
LOGGER.addHandler(ProgressConsoleHandler())
```

Reply:
fixed

Reply:
What about passing a reading/writing flag as a parameter to `LOGGER.info()` on line 116? This would allow reading and writing progress bars to be active simultaneously, e.g. if reading and writing were multithreaded.